app/services/groups/transfer_service.rb

```diff
@@ -47,5 +49,69 @@ def validate(new_namespace)
       raise TransferError,
             I18n.t('services.groups.transfer.same_group_and_namespace')
     end
+
+    def create_activities(old_namespace, new_namespace) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
When transferring a group, the transferred group has two entries for the same action:
![image](https://github.com/user-attachments/assets/29c47f67-3944-436c-8c67-b3b57dd80bde)

Reply:
Fixed in 093ab7b91119f065634ca4b057aba6dbd631abef